Sources_v3/StreamChatUI/ChatChannel/MessageComposer/MessageComposerImageAttachmentCollectionViewCell.swift

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class MessageComposerImageAttachmentCollectionViewCell<ExtraData: ExtraDataTypes>: UICollectionViewCell,
+    AppearanceSetting,
+    UIConfigProvider {
```

Comment:
I think it's worth it to create a common `CollectionViewCell` superclass. I'm sure we will use it a lot.